dotCMS/src/integration-test/java/com/dotcms/rest/api/v3/contenttype/MoveFieldFormTest.java

```diff
@@ -0,0 +1,167 @@
+        assertEquals("The first field data type must be a DataTypes.SYSTEM", RowField.class.cast(fieldLayout.getFields().get(0)).dataType(), DataTypes.SYSTEM);
+        assertEquals("The first field name must be fields-0", "fields-0", RowField.class.cast(fieldLayout.getFields().get(0)).name());
+
+        assertTrue("The second field on the list must be a ColumnField", fieldLayout.getFields().get(1) instanceof ColumnField);
+        assertEquals("The second field data type must be a DataTypes.SYSTEM", ColumnField.class.cast(fieldLayout.getFields().get(1)).dataType(), DataTypes.SYSTEM);
+        assertEquals("The second field name must be fields-1", "fields-1", ColumnField.class.cast(fieldLayout.getFields().get(1)).name());
+
+        assertTrue("The third field on the list must be a TextField", fieldLayout.getFields().get(2) instanceof TextField);
+        assertEquals("The third field data type must be a DataTypes.TEXT", TextField.class.cast(fieldLayout.getFields().get(2)).dataType(), DataTypes.TEXT);
+        assertEquals("The third field name must be text","text", TextField.class.cast(fieldLayout.getFields().get(2)).name());
+
+        assertTrue("The four field on the list must be a TextField", fieldLayout.getFields().get(3) instanceof TextField);
+        assertEquals("The four field data type must be a DataTypes.FLOAT", TextField.class.cast(fieldLayout.getFields().get(3)).dataType(), DataTypes.FLOAT);
+        assertEquals("The four field name must be text", "decimal", TextField.class.cast(fieldLayout.getFields().get(3)).name());
+
+        assertTrue("The five field on the list must be a TextField", fieldLayout.getFields().get(4) instanceof TextField);
+        assertEquals("The five field data type must be a DataTypes.INTEGER",TextField.class.cast(fieldLayout.getFields().get(4)).dataType(), DataTypes.INTEGER);
+        assertEquals("The four field name must be number", "number", TextField.class.cast(fieldLayout.getFields().get(4)).name());
+
+    }
+
+    final List<Map<String, Object>> createTestFields (final String contentTypeId) {
+
+        final List<Map<String, Object>> fieldList = new ArrayList<>();
+        final Map<String, Object> rowField = CollectionsUtils.map("clazz" , "com.dotcms.contenttype.model.field.ImmutableRowField",
+                "contentTypeId", contentTypeId,
+                "dataType" , "SYSTEM",
+                "fieldContentTypeProperties", Collections.emptyList(),
+                "fieldType", "Row",
+                "fieldTypeLabel", "Row",
+                "fieldVariables", Collections.emptyList(),
+                "fixed", Boolean.FALSE,
+                "iDate", new Long(1584647620000l),
+                "id","a2d05a0b-5f4d-449d-9ca2-1117ab1ab086",
+                "indexed",  Boolean.FALSE,
+                "listed", Boolean.FALSE,
+                "modDate", new Long(1584647620000l),
+                "name", "fields-0",
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "name" appears 5 times in this file; the first occurrence is on line 94](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5214528)